MediaPicker/src/main/java/org/wordpress/android/util/WPMediaUtils.java

```diff
@@ -0,0 +1,218 @@
+package org.wordpress.android.util;
+
+import android.app.Activity;
+import android.content.ClipData;
+import android.content.Context;
+import android.content.Intent;
+import android.media.MediaScannerConnection;
+import android.net.Uri;
+import android.os.Environment;
+import android.provider.MediaStore;
+import android.util.Log;
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.appcompat.app.AlertDialog;
+import androidx.core.content.FileProvider;
+import com.google.android.material.dialog.MaterialAlertDialogBuilder;
+import org.wordpress.android.mediapicker.MediaPickerFragment;
+import org.wordpress.android.mediapicker.R;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class WPMediaUtils {
```

Comment:
Related issue: [Convert WPMedtiaUtils to kotlin (#27)](https://github.com/wordpress-mobile/WordPress-MediaPicker-Android/issues/27)